crates/storage/src/schema/revision_0034.rs

```diff
@@ -0,0 +1,47 @@
+use pathfinder_common::StateCommitment;
+use pathfinder_ethereum::EthereumStateUpdate;
+use rusqlite::Transaction;
+
+use crate::{
+    L1StateTable, RefsTable, StarknetBlocksBlockId, StarknetBlocksNumberOrLatest,
+    StarknetBlocksTable,
+};
+
+pub(crate) fn migrate(tx: &Transaction<'_>) -> anyhow::Result<()> {
+    tx.execute("DROP TABLE l1_state", [])?;
+    tx.execute(
+        r"CREATE TABLE l1_state (
+            starknet_block_number      INTEGER PRIMARY KEY,
+            starknet_block_hash        BLOB    NOT NULL,
+            starknet_global_root       BLOB    NOT NULL
```

Comment:
Hmm -- shouldn't we use `starknet_state_commitent` instead of `starknet_global_root`? At least that seems to be the official name now: https://docs.starknet.io/documentation/architecture_and_concepts/State/starknet-state/#state_commitment